### PR TITLE
React-Portfolio-Lisa J

### PR DESCRIPTION
**Pull request by LisaVasic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,8 @@
 # React Portfolio
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
-
+The asignment was to remake my portfolio using React and styled components. I was supposed to use my own json file to fecth data and display my featured projects. I was following a given design.
```
Comment (MariaWellander) [inline README.md:2]: Clear description - super!

File: README.md [hunk 2]
```diff
@@ -1,10 +1,8 @@
...
 ## The problem
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+For this project I made a structure and split the site up in 9 components. 
+If I had more time I would spend more time planning the styling and structure to make the code neater and the styling more consistent.
 
 ## View it live
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://portfolio-lisa-johansson.netlify.app/
```
Comment (MariaWellander) [inline README.md:8]: Nice that you changed the Netlify URL - sleek!

File: package.json [hunk 3]
```diff
@@ -11,7 +11,8 @@
     "eslint-plugin-react": "^7.30.1",
     "eslint-plugin-react-hooks": "^4.6.0",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "styled-components": "^5.3.6"
```
Comment (MariaWellander) [inline package.json:15]: Good start! Lovely to have the new package for the week installed :D

File: src/App.js [hunk 4]
```diff
@@ -1,9 +1,27 @@
...
+      <Header />
+      <Intro />
+      <Tech />
+      <Projects />
+      <OtherProjects />
+      <Thoughts />
+      <Skills />
+      <ForMore />
+      <Contact />
+    </>
```
Comment (MariaWellander) [inline src/App.js:24]: Tidy - nice!

File: src/FeaturedProject.json [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+      "image": "/Assets/project-thumbnails/mapty.PNG"
+    },
+    {
+      "title": "Music Releases",
+      "title-description": "4 news site about avocados",
+      "project-description": "A responsive web site using HTML & CSS",
+      "repo-name": "project-news-site",
+      "netlify-name": "avocodo-news",
+      "tags": ["HTML5", "CSS3"],
+      "image": "/Assets/project-thumbnails/guessmynumber.PNG"
```
Comment (MariaWellander) [inline src/FeaturedProject.json]: Would be wonderful if we could update the content here for these 3 projects :) Super that you made it work for the first one!

File: src/components/Contact.js [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+export const ContactOuterWrapper = styled.div`
+  width: 100%;
+  height: 300px;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  align-items: center;
+`;
```
Comment (MariaWellander) [inline src/components/Contact.js:23]: Congrats to the first use of styled components!

File: src/components/ForMore.js [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+import React from 'react';
+import styled from 'styled-components';
+import heroLinkedin from '../assets/linkedin_icon.png';
+import heroGithub from '../assets/github_icon.png';
+import heroStackOverflow from '../assets/stackoverflow_icon.png';
+
+export const ForMore = () => {
+  return (
+    <ForMoreOuterWrapper>
+      <ForMoreInnerWrapper>
```
Comment (MariaWellander) [inline src/components/ForMore.js:11]: Ah! The outer- and innerwrappers for each component that you talked about during the demo - neat and organized to used them like this :)

File: src/components/ForMore.js [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+    align-items: center;
+    white-space: nowrap;
+    
+    a {
+      text-decoration:none;
+    }
+
+    :hover {
+      border: solid var(--white);
+    }
```
Comment (MariaWellander) [inline src/components/ForMore.js:100]: Very sleek impression with this clean hover effect!

File: src/components/Header.js [hunk 9]
```diff
@@ -0,0 +1,183 @@
...
+    border-radius: 50%;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+  }
+
+  a:hover{
+    background-color: rgb(41, 58, 79);
+  }
```
Comment (MariaWellander) [inline src/components/Header.js:77]: Also a very sleek and clean hover effect. Looks like the same color as the h5 background - nice from a design perspective :)

File: src/components/OtherProjects.js [hunk 10]
```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export const OtherProjects = () => {
+  return (
+    <OtherProjectOuterWrapper>
+      <OtherProjectInnerWrapper>
+        <h6 className="otherProjects">OTHER PROJECTS</h6>
+        <a href="INSERT-LINK-TO-PROJECTS-NETLIFY">
```
Comment (MariaWellander) [inline src/components/OtherProjects.js:9]: Would be wonderful to have the Weather App link here, to view the project live :D

File: src/components/Projects.js [hunk 11]
```diff
@@ -0,0 +1,165 @@
...
+                  {project.tags.map((tag) => (
+                    <ProjectTags>
+                      {tag}
+                    </ProjectTags>
+                  ))}
+                </ul>
+              </ProjectCard>
+            );
+          })}
+        </FeaturedProjectWrapper>
```
Comment (MariaWellander) [inline src/components/Projects.js:37]: Wonderful work with the first featured project, Movies! Though, when clicking on the other 3 projects I'm ending up on wrong pages. Looks like some changes needs to be done in the json :)

File: src/components/Skills.js [hunk 12]
```diff
@@ -0,0 +1,99 @@
...
+    display:none;
+}
+
+@media (min-width: 668px) {
+    gap: 30px;
+
+    .More {
+        display:inline;
+        flex-direction: column;
+    }
```
Comment (MariaWellander) [inline src/components/Skills.js:79]: Cool that you added the More column here - just as the CCP design :) Though, CCP also has a column here called Upcoming. Donnu if it's super crucial not to have that one, probably not but thought I should mention it :)

File: src/index.css [hunk 13]
```diff
@@ -11,3 +11,41 @@ code {
...
+    font-size:25px;
+  }
+  
+ }
+
+ @media (min-width: 1024px) {
+   h5 {
+    font-size: 27px;
+   }
+  }
```
Comment (MariaWellander) [inline src/index.css:51]: Nice to also use CSS :)

**Review by MariaWellander — COMMENTED** [review]
**Am I able to understand the code easily? Does the code have lots of duplicates? Could it be tidied up?**

I'm absolutely able to understand the code easily - good job with that. It looks really structured and tidy. Maybe one could erase some duplicates but I'm not the one to instruct on how :)

**Does it work or does anything look broken?**

It works! Though, when clicking on the 3 last featured projects I'm ending up on the wrong page. I made a comment about it in the code with an answer to how to solve that issue - hope that's the right solution :D Very nice otherwise! Works perfectly fine using different devices and clicking all the icons.

**Does it follow the general and blue requirements?**

Yes, it does! Though, in the instructions we were told to add the personal thoughts about code and that is missing (Latin text right now). It's not a blue requirement but I thought I should mention it :) Looking forward to read it - your texts are always so comprehensive and on the spot, Lisa! Be proud of that :D

**To sum it up**

Wonderful work, Lisa! You should be very proud of yourself :D High five!
